tests/ui/trait-bounds/bad-suggestionf-for-repeated-unsized-bound-127441.stderr

```diff
@@ -3,36 +3,54 @@ error[E0203]: type parameter has more than one relaxed default bound, only one i
    |
 LL | fn foo2<T: ?Sized + ?Sized>(a: T) {}
    |            ^^^^^^   ^^^^^^
+   |
+   = help: add `#![feature(more_maybe_bounds)]` to the crate attributes to enable
```

Comment:
This shouldn't be allowed, even w/ the feature gate.

Reply:
`fn foo<T: Clone + Clone>() {}` is supported though, it doesn't even warn.